server/todo/list.go

```diff
@@ -1,4 +1,4 @@
-package main
+package todo
```

Comment:
This discussion might be out of scope of the PR:

I've noticed that most plugin have common packages like `command` and `api` which interface with a plugin specific package (in this case `todo`) To unity the naming even more, it could help to also unify the naming of that package e.g. into `app`.

Thoughts @iomodo @levb ?